services/order.py

```diff
@@ -0,0 +1,39 @@
+from datetime import datetime
+from typing import Dict
+
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, User, Ticket
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[Dict],
+        username: str,
+        date: datetime = None
+) -> None:
+    order = Order.objects.create(
+        user=User.objects.get(username=username)
+    )
+    if date:
+        order.created_at = date
+        order.save()
+
+    for ticket in tickets:
+        Ticket.objects.create(
+            movie_session_id=ticket["movie_session"],
+            row=ticket["row"],
+            seat=ticket["seat"],
+            order=order
+        )
+
+
+def get_orders(
+        username: str = None
+) -> QuerySet[Order]:
+    orders = Order.objects.all()
+    if username:
+        user = User.objects.get(username=username)
```

Comment:
The 'get_orders' function uses 'User.objects.get', which will raise a 'DoesNotExist' exception if the username is not found. Consider using 'get_object_or_404' or handling the exception to improve robustness.